Utilities/bootstrap

```diff
@@ -597,9 +597,9 @@ def main():
         embed_rpath = "$ORIGIN/../lib/swift/linux"
     else:
         embed_rpath = "@executable_path/../lib/swift/macosx"
-    cmd.extend(["-Xlinker", "-rpath", "-Xlinker", embed_rpath])
 
     cmd = [os.path.join(sandbox_path, "bin", "swift-build")]
```

Comment:
Nitpicking here, but I would prefer that we move this line before the if-statement instead.